### ISSUE TITLE
Date and time display in metabox

### ISSUE THREAD

**Issue by benoitchantre** [stuttter/wp-event-calendar#14]
It would be great to have a filter switch from `mm/dd/yyyy` in metaboxes to `dd/mm/yyyy`.
Same thing for time to switch to 24 hours format.

**Comment by leoauri** [comment]
Absolutely, possibly link it to the locale.  Very disconcerting if you don't think dates in that direction

**Comment by knutsp** [comment]
Add filter for the metabox date, as I prefer 'd.m.Y' (PHP) and 'dd.mm.YY' (jQuery Datapicker). Also remove am/pm display and adjustments when 24h clock.
